### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by dalia-nasr**

### CODE REVIEW

File: src/styles/functions.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const growOverParentPadding = (widthPercent) => css`
  * @property {function=} extraLarge
  */
 /** @type {mediaQueries} */
-const media = Object.keys(mediaSizes).reduce((acc, label) => {
+const Media = Object.keys(mediaSizes).reduce((acc, label) => {
   if (label === 'extraLarge') {
     acc[label] = (...args) => css`
```
Comment (iamzoltan) [inline src/styles/functions.js]: this change in variable name breaks something in our base components container file.

**Review by iamzoltan — CHANGES_REQUESTED** [review]
looks good, everything is running in the new version of gatsby and node,  some minor tweaks and we can close this PR.

is there a reason you removed the camel case?

File: src/styles/functions.js (hunk 1 above)
Comment (dalia-nasr) [reply src/styles/functions.js]: It's due to naming convention warnings. Do you prefer to return them to the previous namings, or I update the code according to the new ones?

File: src/pages/404.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const Container = styled.div`
   text-align: center;
 `;
```
Comment (iamzoltan) [inline src/pages/404.js:9]: one word

File: src/pages/about/finances.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Finances = () => {
```
Comment (iamzoltan) [inline src/pages/about/finances.js:32]: one word, should be lowercase

File: src/pages/about/mission.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Mission = () => {
```
Comment (iamzoltan) [inline src/pages/about/mission.js:33]: one word

File: src/pages/about/policies.js [hunk 5]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Policies = () => {
 
   return (
     <Layout>
```
Comment (iamzoltan) [inline src/pages/about/policies.js:35]: one word

File: src/pages/about/press.js [hunk 6]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Press = () => {
```
Comment (iamzoltan) [inline src/pages/about/press.js:32]: should be lower case

File: src/pages/cookie.js [hunk 7]
```diff
@@ -41,7 +41,7 @@ const StyledTable = styled.table`
   }
 `;
 
-export default () => (
+const Cookie = () => (
```
Comment (iamzoltan) [inline src/pages/cookie.js:44]: one word

File: src/pages/donate/give.js [hunk 8]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Give = () => {
```
Comment (iamzoltan) [inline src/pages/donate/give.js:32]: one word

File: src/pages/donate/partners.js [hunk 9]
```diff
@@ -30,7 +30,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Partners = () => {
```
Comment (iamzoltan) [inline src/pages/donate/partners.js:33]: one word

File: src/pages/index.js [hunk 10]
```diff
@@ -85,7 +85,7 @@ TopicHeading.propTypes = {
 //`;
 
 // -- MAIN
```
Comment (iamzoltan) [inline src/pages/index.js:87]: one word

File: src/pages/join/careers.js [hunk 11]
```diff
@@ -29,7 +29,7 @@ TopicHeading.propTypes = {
 
 
 // -- MAIN
-export default () => {
+const Careers = () => {
```
Comment (iamzoltan) [inline src/pages/join/careers.js:32]: one word

File: src/pages/resources/job-board.js [hunk 12]
```diff
@@ -100,11 +100,11 @@ TopicHeading.propTypes = {
 };
 
 // -- MAIN
-export default () => {
+const JobBoard = () => {
```
Comment (iamzoltan) [inline src/pages/resources/job-board.js:103]: two word, first should be lower case

File: src/pages/join/volunteering.js [hunk 13]
```diff
@@ -100,7 +100,7 @@ TopicHeading.propTypes = {
 };
 
 // -- MAIN
```
Comment (iamzoltan) [inline src/pages/join/volunteering.js:102]: all lowercase

File: src/pages/terms-of-use.js [hunk 14]
```diff
@@ -16,7 +16,7 @@ const BrandFont = styled.span`
   }
 `;
 
-export default () => (
+const TermsOfUse = () => (
```
Comment (iamzoltan) [inline src/pages/terms-of-use.js:19]: first should be lower case

File: src/styles/basedStyles.js [hunk 15]
```diff
@@ -65,9 +65,10 @@ const scrollStyle = css`
   }
 `;
 
-export default {
+ const basedStyle ={
```
Comment (iamzoltan) [inline src/styles/basedStyles.js:68]: looks good!

File: src/styles/globalStyles.js [hunk 16]
```diff
@@ -26,9 +26,11 @@ const LockScrollWithVerticalControl = createGlobalStyle`
   }
 `;
 
-export default {
+const globalStylesExport = {
```
Comment (iamzoltan) [inline src/styles/globalStyles.js:29]: looks good!

File: src/utils/color.js [hunk 17]
```diff
@@ -224,11 +224,12 @@ function transparentize(colorString, alpha = 1) {
   });
 }
 
-export default {
+ const Colors = {
```
Comment (iamzoltan) [inline src/utils/color.js:227]: one word

File: src/utils/common.js [hunk 18]
```diff
@@ -51,10 +51,11 @@ const scrollBy = (top = 0, left = 0) => {
   }
 }
 
-export default {
+ const Common = {
```
Comment (iamzoltan) [inline src/utils/common.js:54]: one word, lowercase

File: src/utils/fake.js [hunk 19]
```diff
@@ -56,10 +56,11 @@ function generateFakeMatches(numOfMatches) {
   };
 }
 
-export default {
+ const Fake ={
```
Comment (iamzoltan) [inline src/utils/fake.js:59]: lowercase

File: src/utils/logger.js [hunk 20]
```diff
@@ -25,6 +25,7 @@ function error(funcName, msg) {
   throw new Error(`[${funcName}] ${msg}`);
 }
 
-export default {
+ const Logger ={
```
Comment (iamzoltan) [inline src/utils/logger.js:28]: lowercase

File: src/utils/reactSelectHelpers.js [hunk 21]
```diff
@@ -18,7 +18,8 @@ const saveFormatToOptions = (val) => {
   return { value: val, label: val };
 };
 
-export default {
+ const ReactSelectShape ={
```
Comment (iamzoltan) [inline src/utils/reactSelectHelpers.js:21]: first word should be lowercase

File: src/utils/timePickerHelpers.js [hunk 22]
```diff
@@ -37,7 +37,8 @@ const deserializeSelectedDatetime = serializedDatetimeString =>
       }, [])
     : []
```
Comment (iamzoltan) [inline src/utils/timePickerHelpers.js:39]: first word should be lowercase

**Review by iamzoltan — CHANGES_REQUESTED** [review]

**Review by iamzoltan — APPROVED** [review]

**Pull request merged**